This removes the behavior of refreshing all Quagga daemons when any interfaces
go up or down. This dependency was added as a custom feature for the bundled
version of Solaris and does not exist in the upstream version, so it will not
be submitted upstream.

--- solaris/quagga.xml.in
+++ solaris/quagga.xml.in
@@ -53,7 +53,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
@@ -176,7 +176,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
@@ -299,7 +299,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
@@ -422,7 +422,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
@@ -548,7 +548,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
@@ -669,7 +669,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
@@ -805,7 +805,7 @@
 	<dependency
 		name='network_routing_setup'
 		grouping='require_all'
-		restart_on='refresh'
+		restart_on='none'
 		type='service'>
 		<service_fmri value='svc:/network/routing-setup' />
 	</dependency>
